TombEngine/Game/control/trigger.cpp

```diff
@@ -118,7 +118,7 @@ int GetSwitchTrigger(ItemInfo* item, short* itemNos, int attatchedToSwitch)
 	return 0;
 }
 
-bool SwitchTrigger(short itemNumber, short timer)
+int SwitchTrigger(short itemNumber, short timer)
```

Comment:
I've seen this in other code files but how about use casting from bool to int? because now if you have "return 1". I mean ik it's obvious but it's for readiness.

Reply:
Yes, that's one of the things we will do, but Raildex suggest to do that in a different branch, so I unmade the bool update.